source/Projectile.cpp

```diff
@@ -35,7 +35,7 @@ namespace {
 
 
 
-Projectile::Projectile(const Ship &parent, Point position, Angle angle, const Weapon *weapon)
+Projectile::Projectile(const Ship &parent, const Point &position, const Angle &angle, const Weapon *weapon)
```

Comment:
The underlying Body constructor takes its point and angle parameters by value. While we don't quite forward these by-value parameters, a follow-up (profiled) PR can consider changing this constructor.
```suggestion
Projectile::Projectile(const Ship &parent, Point position, Angle angle, const Weapon *weapon)
```